cigaradvisor/scripts/linking-data.js

```diff
@@ -91,4 +134,5 @@ export default function addLinkingData() {
     addOrUpdateCollection();
     window.addEventListener('hashchange', addOrUpdateCollection);
   }
+  addFAQLdJson();
```

Comment:
Should this run on every page, or should it only run on blog posts?

Reply:
This should only run on blog pages. I tried placing the function call in other places, but I was running into an issue where the content didn't exist on the page when it executed. Is there a better location to put this, or put it in an if statement? What can I check to determine if it is a blog post?

Reply:
you could do:

```suggestion
  else if (getCategory(window.location.pathname)) {
    addFAQLdJson();
  }
```

If the page is a category page, the previous if will run, otherwise, this would return something and the page would be within a category folder, making it a blog post.

Reply:
Ok, I made the suggested change and retested. Look good. Thanks.